ruby_on_rails/app_initialisation.md

```diff
@@ -13,7 +13,11 @@
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
 You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
+* Start a new Rails project using
+```
+rails new [project-name] --database=postgresql --skip_ci --no-skip-test --skip_action_mailbox -m https://raw.githubusercontent.com/renuo/applications-setup-guide/main/ruby_on_rails/template.rb
```

Comment:
Also the link does not work at the moment because the template file hasn't been pushed yet but let's wait and seen.